app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,53 @@
+module SessionsHelper
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    if user_id = session[:user_id]
+      @current_user ||= User.find_by(id: user_id) if session[:user_id]
```

Comment:
Toán tử này có tác dụng gì em `||=`

Reply:
Toán tử `||=` có tác dụng là, giả sử ta có `a ||= b` thì nếu a không xác định hoặc thuộc kiểu `falsy` thì ta sẽ gán `a = b`, còn nếu như `a` đã được xác định rồi thì ta không cần thực hiện phép gán `a = b` nữa.
Như ở dòng lệnh `@current_user ||= User.find_by(id: user_id)` thì ta thấy ta sẽ gán biến instance `@curent_user` cho user hiện tại được tìm kiếm qua id, còn nếu như `@current_user` đã có chứa người dùng rồi thì không cần gán nữa.